### ISSUE TITLE
Two versions of Control Center

### ISSUE THREAD

**Issue by Almansherov** [subutai-io/control-center#1077]
There is a threat that users in the future might get confused and scared with all those terminals and code lines. 
Therefore, there is way to make two versions of Control Central: 
1) version for the user with no-technical background (all the processes would be done only on the background with no terminals open.
2) version for the users with technical background. 
So, is there a need for us to implement it?

**Comment by akarasulu** [comment]
I would not make this a two version thing but make it modal. You can hide things in the background and have a setting to control that. By default things can be hidden to keep things simple and not confuse users. The option can be there to enable it in the foreground where users can see them like the peer building terminal etc.

**Comment by ux-scott** [comment]
Agreed, one version. I would so that the default state (the "simple" version) shows steps in the process, progress bars, countdown timer if possible, so that user knows that things are moving forward, is informed of progress, does not panic if it takes too long. Then an "advanced" control would expand the dialog to show more info - not sure if this is possible, but is there something like a terminal viewer where they could watch the action scroll past, and scroll back up to check things if needed?

**Comment by Almansherov** [comment]
@Erkin97 what do you think?

**Comment by ux-scott** [comment]
I've sketched out that idea. It's in the updated Axure link I just sent to you on Slack, Alex. (The second link, for some reason.)

**Comment by Almansherov** [comment]
Discussion is closed.

**Comment by Almansherov** [comment]
Some comments from Slack:

Also this Components thing should change where it is modal: (1) current implementation is advanced mode, (2) default simple mode should show capabilities that you can install to have groups of things like (web client, desktop client, peer management). Checking off one will install the set of things needed for it. This will be much easier for users.
